### PR TITLE
update readme examples with correct API, URLs and package-name

### LINKED ISSUE #230: Update README exmples with import statements for clarity
The API usage in the readme-examples are not clearly spelled out where to import them from. Additionally `calibration` should be `Calibration` in the redme-example. This PR fixes these issues and makes the readme-examples reflect the current-API and PyPI package name.

### PR DESCRIPTION
**Pull request by sugatoray**
# What does this PR do?

The API usage in the readme-examples are not clearly spelled out where to import them from. Additionally `calibration` should be `Calibration` in the redme-example. This PR fixes these issues and makes the readme-examples reflect the current-API and PyPI package name.

Fixes #230 (issue)

## Before submitting
- [x] Did you read the [contributor guideline](https://github.com/huggingface/optimum-quanto/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you run all tests locally and make sure they pass. `NA`
- [ ] Did you write any new necessary tests? `NA`

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by sugatoray** [comment]
This change is also tied to the fixes proposed in the following PR for Quanto-Blog:

https://github.com/huggingface/blog/pull/2190

File: README.md [hunk 1]
```diff
@@ -112,6 +112,13 @@ Optimum Quanto is available as a pip package.
 pip install optimum-quanto
 ```
 
+> :bulb: **NOTE**: `optimum-quanto` was initially released under the 
+> name [`quanto`](https://pypi.org/project/quanto) on PyPI. However, 
+> it was later rebranded under the name 
+> [`optimum-quanto`](https://pypi.org/project/optimum-quanto) on PyPI 
+> and on the GitHub repository. So, if you are using this library, 
+> please use the `optimum-quanto` package and NOT `quanto` from PyPI. 
+
```
Comment (dacorvo) [inline README.md:121]:
```suggestion
```

**Review by dacorvo — CHANGES_REQUESTED** [review]
Thank you for this pull-request. The imports had been omitted for the sake of simplicity, but I don't mind adding them (and qint8 was already imported).
You need to use conventional commits in your pull-request, otherwise it cannot be merged.

**Comment by dacorvo** [comment]
Closing as obsoleted by #243

**Pull request closed without merging**